### PR TITLE
ttl: make the default value of `ttl_job_interval` to `24h`

### LINKED ISSUE #57757: make the default value of `ttl_job_interval` to a larger value
## Enhancement

If a user does not specify the `ttl_job_interval` when creating a TTL table, it will use `1h` as the default value. However, this default value seems to make the job runs too frequently and have some performance effect on the cluster. It's better to make the default value to `24h` to avoid this.

### PR DESCRIPTION
**Pull request by lcwangchao**
### What problem does this PR solve?

Issue Number: close #57757

### What changed and how does it work?

The default value of `ttl_job_interval` changed from `1h` to `24h`

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
ttl: The default value of `ttl_job_interval` changed from `1h` to `24h`
```

### CODE REVIEW

File: pkg/ddl/ttl.go [hunk 1]
```diff
@@ -30,9 +30,6 @@ import (
 	"github.com/pingcap/tidb/pkg/util/dbterror"
 )
 
-// DefaultTTLJobInterval is the default value for ttl job interval.
-const DefaultTTLJobInterval = "1h"
```
Comment (lcwangchao) [inline pkg/ddl/ttl.go:34]: This const is moved to `meta` package:  https://github.com/pingcap/tidb/pull/57758/files#diff-1730439e8c1b7d1a7019391006a2c0cb2f455b9de48769c9b6097c9c0a2b995eR1324

**Comment by D3Hunter** [comment]
/approve

ddl/meta pkg lgtm

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**